### PR TITLE
Feature group-dashboard-expanse-friends

### PR DESCRIPTION
**Pull request by 1810Nikita**
*Linked Work Items*
https://internship-splitwise.atlassian.net/jira/software/projects/IS/issues/IS-84
*Checklist*
- [x] Code adheres to *[Git and Pull Request Guidelines]*
- [x] Code adheres to *[Angular Guidelines]*
- [x] Code adheres to *[Front-end Checklist]*

### CODE REVIEW

**Comment by AshaPatel216** [comment]
Spelling mistake in commit message:
Create the home page for aplication -> Create the home page for application

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,8 +1,19 @@
-<!--First page of splitwise application where you can find option for login to your account and use aplication-->
-<div class="text-center">
-    <!--Heading-->
-    <h1>Welcome to Splitwise</h1>
-    <!--Button for routing to the login page-->
-    <button type="button" class="login" *ngIf="showLoginButton" routerLink="/login-signup/login"><i class='fas fa-user-graduate' id="graduate">Login to your Account</i></button>
+<div class="app-body">
```
Comment (AshaPatel216) [inline src/app/app.component.html:1]: No comments in html

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+    <nav class="navbar sticky-top navbar-light" style="background-color:#17179a;">
```
Comment (AshaPatel216) [inline src/app/app.component.html]: inline style should be avoided

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
+        <a class="navbar-brand"><h1>Splitwise</h1></a>
+        <div class="text-end"  id= "logout-button">
+            <button type="button" class="btn btn-outline-light btn-sm" *ngIf="showsidenav" routerLink="/">Logout</button>
```
Comment (AshaPatel216) [inline src/app/app.component.html]: Wrong naming convention of variable name showsidenav

File: src/app/app.component.html [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
+        </div>
+    </nav>  
+    <div *ngIf="showLogin">
+        <app-login></app-login>
+    </div>   
+    
+    
+    <div class="sidenav">
```
Comment (AshaPatel216) [inline src/app/app.component.html:13]: wrong naming conventions

File: src/app/app.component.html [hunk 5]
```diff
@@ -1,8 +1,19 @@
...
+        <app-sidenav (onToggleSidenav)="onToggleSidenav($event)" *ngIf="showsidenav"></app-sidenav>
```
Comment (AshaPatel216) [inline src/app/app.component.html]: same as above

File: src/app/app.component.scss [hunk 6]
```diff
@@ -1 +1,48 @@
+@import 'src/app/styles/layout/variable.scss';
+/* Adjust logo size for smaller screens */
+.sidenav {
+  background: $background-color ;
+  overflow: hidden;
+}
+.navbar-brand {
+  font-size: 32px; /* changed from 2rem to 32px */
+  margin-bottom: 0;
+  color: #ffff;
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: please check guideline to use color code. #ffff -> #fff

File: src/app/app.component.scss [hunk 7]
```diff
@@ -1 +1,48 @@
...
+}
+
+@media (max-width: 767px) {
+  .logo {
+    font-size: 24px; /* changed from 1.5rem to 24px */
+  }
+}
+
+/* Adjust menu icon size for smaller screens */
+.menu_icon {
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Inconsistency in class name. Follow hyphen separated values only

File: src/app/app.component.scss [hunk 8]
```diff
@@ -1 +1,48 @@
...
+  font-size: 24px; /* changed from 1.5rem to 24px */
+}
+
+@media (max-width: 767px) {
+  .menu_icon {
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Same as above

File: src/app/app.component.scss [hunk 9]
```diff
@@ -1 +1,48 @@
...
+    font-size: 20px;
+  }
+}
+
+/* Adjust menu button size for smaller screens */
+.menu button {
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Follow SCSS rule

File: src/app/app.component.scss [hunk 10]
```diff
@@ -1 +1,48 @@
...
+  font-size: 14px;
+  padding: 3px 6px;
+}
+
+@media (max-width: 767px) {
+  .menu button {
+    font-size: 12px;
```
Comment (AshaPatel216) [inline src/app/app.component.scss:39]: Follow SCSS rule

File: src/app/app.component.ts [hunk 11]
```diff
@@ -9,17 +16,56 @@ import { Router } from '@angular/router';
 export class AppComponent {
   title = 'splitwise';
   // Declare showLoginButton property
-  showLoginButton: boolean;
+  showLogin: boolean;
+  isSideNavCollapsed = false;
```
Comment (AshaPatel216) [inline src/app/app.component.ts]: Again variable initialization and declaration rule not followed

File: src/app/app.component.ts [hunk 12]
```diff
@@ -9,17 +16,56 @@ import { Router } from '@angular/router';
...
-        this.showLoginButton = false;
+        this.showLogin = false;
       }
     });
+
+    
+    this.showsidenav = false;
+    this.router.events.subscribe((val) => {
+      if (this.router.url === '/' || this.router.url == '/login-signup/register' || this.router.url == '/login-signup/login') {
+        this.showsidenav = false;
```
Comment (AshaPatel216) [inline src/app/app.component.ts]: Follow naming conventions

File: src/app/app.component.html (hunk 1 above)
Comment (1810Nikita) [reply src/app/app.component.html:1]: Change done.

File: src/app/app.component.html (hunk 2 above)
Comment (1810Nikita) [reply src/app/app.component.html]: Change done.

File: src/app/app.component.html (hunk 3 above)
Comment (1810Nikita) [reply src/app/app.component.html]: Change done.

File: src/app/app.component.html (hunk 5 above)
Comment (1810Nikita) [reply src/app/app.component.html]: Change done.

File: src/app/app.component.scss (hunk 6 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: change done.

File: src/app/app.component.scss (hunk 7 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.scss (hunk 8 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.scss (hunk 9 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.ts (hunk 11 above)
Comment (1810Nikita) [reply src/app/app.component.ts]: Change done.

File: src/app/app.component.ts (hunk 12 above)
Comment (1810Nikita) [reply src/app/app.component.ts]: Change done.

File: src/app/app.component.scss [hunk 13]
```diff
@@ -1 +1,53 @@
...
+  overflow: hidden;
+}
+.navbar-brand {
+  font-size: 32px;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+#navbar {
+  background-color: #24247e;
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Use variable for color

File: src/app/app.component.scss [hunk 14]
```diff
@@ -1 +1,53 @@
+@import 'src/app/styles/layout/variable.scss';
+//side nav
+.sidenav {
+  background: $background-color ;
+  overflow: hidden;
+}
+.navbar-brand {
+  font-size: 32px;
+  margin-bottom: 0;
+  color: #fff;
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Use variable for color

File: src/app/app.component.scss [hunk 15]
```diff
@@ -1 +1,53 @@
+@import 'src/app/styles/layout/variable.scss';
+//side nav
+.sidenav {
+  background: $background-color ;
+  overflow: hidden;
+}
+.navbar-brand {
+  font-size: 32px;
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Use variable for font size

File: src/app/app.component.scss (hunk 13 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.scss (hunk 14 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.scss (hunk 15 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Change done.

File: src/app/app.component.ts [hunk 16]
```diff
@@ -9,17 +16,48 @@ import { Router } from '@angular/router';
...
+        this.showSideNav = true;
+      }
+    });    
   }
-  
+
+  ngOnInit(): void { 
+  }
+
+  onToggleSidenav(data: SideNavToggle): void {
```
Comment (AshaPatel216) [inline src/app/app.component.ts:62]: Missing comments

File: src/app/app.component.ts [hunk 17]
```diff
@@ -9,17 +16,48 @@ import { Router } from '@angular/router';
...
+    this.screenWidth =data.screenWidth;
+    this.isSideNavCollapsed = data.collapsed;  
+  }
+
+  urlStartsWith(path : string) : boolean {
```
Comment (AshaPatel216) [inline src/app/app.component.ts:74]: Same as above

File: src/app/app.component.ts (hunk 16 above)
Comment (1810Nikita) [reply src/app/app.component.ts:62]: Changes done.

File: src/app/app.component.ts (hunk 17 above)
Comment (1810Nikita) [reply src/app/app.component.ts:74]: Changes done.

File: src/app/body/body.component.html [hunk 18]
```diff
@@ -0,0 +1,4 @@
+<!--this is the body for app-component it is dispalying the component after routing-->
```
Comment (AshaPatel216) [inline src/app/body/body.component.html]: check spellings

File: src/app/app.component.scss [hunk 19]
```diff
@@ -1 +1,53 @@
...
+    font-size: 24px; 
+  }
+}
+
+/* Adjust menu icon size for smaller screens */
+.menu-icon {
+  font-size: 24px; 
+}
+
+@media (max-width: 767px) {
```
Comment (AshaPatel216) [inline src/app/app.component.scss:31]: duplicate media query

File: src/app/app.component.scss [hunk 20]
```diff
@@ -1 +1,53 @@
...
+  }
+}
+
+/* Adjust menu button size for smaller screens */
+.menu-button {
+  font-size: 14px;
+  padding: 3px 6px;
+}
+
+@media (max-width: 767px) {
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: duplicate media query

File: src/app/app.component.scss [hunk 21]
```diff
@@ -1 +1,53 @@
...
+  .menu-button {
+    font-size: 12px;
+    padding: 2px 4px;
+  }
+}
+
+#logout-button {
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: why this style is at the end of media query? this should be before media query

File: src/app/dashboard/dashboard.component.html [hunk 22]
```diff
@@ -0,0 +1,22 @@
+<div class="dashboard">
+    <!-- Dashboard-->
+    <nav class="navbar navbar-light" style="background-color:#f0f8ff;">
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: Avoid inline style

File: src/app/dashboard/dashboard.component.html [hunk 23]
```diff
@@ -0,0 +1,22 @@
...
+        <a class="navbar-brand"><h1>Dashboard</h1></a>
+        <!--Buttons for expense and settle up-->
+        <div class="d-grid gap-2 d-md-flex justify-content-md-end m-2">
+            <button class="btn btn-primary me-md-2" id="navbar-button" type="button" routerLink="/expense/add-expense">Add Expense</button>
+            <button class="btn btn-success" id="navbar-button" type="button" routerLink="/settle-up">Settle Up</button>
+        </div>
+    </nav>
+    <!-- all balance grid -->
+    <div class="grid-container1" id="all-balance">
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: class name not proper avoid 1,2,3

File: src/app/dashboard/dashboard.component.html [hunk 24]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="grid-item1">Total balance</div>
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: same as above

File: src/app/dashboard/dashboard.component.scss [hunk 25]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    .grid-item {
+        border: 1px solid $primary-color;
+        padding: 20px;
+        background-color: $secondary-color;
+        height: 400px;
+    }
+    
+    @media only screen and (max-width: 600px) {
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.scss]: Media queries should at the end of normal style

File: src/app/dashboard/dashboard.component.scss [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+//style for all transaction
+.grid-container1 {
+    display: grid;
+    margin:2px ;
+    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    gap: 2px;
+}
+   
+.grid-item1 {
+    background-color: #f8ecf6;
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.scss]: variable for color code

File: src/app/expenses/expense-list/expense-list.component.html [hunk 27]
```diff
@@ -1 +1,45 @@
-<p>expense-list works!</p>
+<!-- This container holds the list of groups -->
+<div class="expense-list">
+  <div class="container" style="overflow-x:auto;">
```
Comment (AshaPatel216) [inline src/app/expenses/expense-list/expense-list.component.html]: Avoid inline style

File: src/app/friend/friend-list/friend-list.component.scss [hunk 28]
```diff
@@ -0,0 +1,10 @@
+@import 'src/app/styles/layout/variable.scss';
+
+//list style
+.expense-list {
```
Comment (AshaPatel216) [inline src/app/friend/friend-list/friend-list.component.scss]: Duplicate style of expanse list if style is common then it should ne common scss file which is accessible to all components.

File: src/app/expenses/expense-list/expense-list.component.ts [hunk 29]
```diff
@@ -7,4 +9,72 @@ import { Component } from '@angular/core';
...
+    const amountOwedPerPerson = totalAmount / numPeople;
+    const amountOwedPerPersonObj: { [key: string]: number } = {};
+    for (const person of expense.people) {
+      amountOwedPerPersonObj[person] = amountOwedPerPerson;
+    }
+    return amountOwedPerPersonObj;
+  }
+
+  //edit will navigate to add expense form
+  editExpense(expense: Expense): void {
```
Comment (AshaPatel216) [inline src/app/expenses/expense-list/expense-list.component.ts:75]: Inconsistency in methos comments

File: src/app/expenses/expense-list/expense-list.component.ts [hunk 30]
```diff
@@ -7,4 +9,72 @@ import { Component } from '@angular/core';
...
+    this.router.navigate(['/expense/add-expense']);
+  }
+
+  //delete will remove the data 
+  deleteExpense(expense: Expense): void {
+    const index = this.expenses.indexOf(expense);
```
Comment (AshaPatel216) [inline src/app/expenses/expense-list/expense-list.component.ts:85]: Inconsistency in methos comments

File: src/app/friend/add-friend/add-friend.component.ts [hunk 31]
```diff
@@ -0,0 +1,56 @@
...
+  // This method is called when the component is initialized
+  ngOnInit(): void {
+    // Initialize the inviteForm FormGroup with a FormArray of names and a message
+    this.inviteForm = this.fb.group({
+      names: this.fb.array([this.createNameInput()]),
+      message: ['']
+    });
+  }
+
+  // This method returns a new FormGroup with a name field
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts]: Inconsistency in methos comments

File: src/app/friend/add-friend/add-friend.component.ts [hunk 32]
```diff
@@ -0,0 +1,56 @@
...
+  createNameInput(): FormGroup {
+    return this.fb.group({
+      name: ['']
+    });
+  }
+
+  // This method returns the FormArray of names from the inviteForm FormGroup
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts]: Inconsistency in methos comments

File: src/app/friend/add-friend/add-friend.component.ts [hunk 33]
```diff
@@ -0,0 +1,56 @@
...
+  deleteNameInput(index: number): void {
+    this.names.removeAt(index);
+  }
+
+  // This method is called when the user clicks the close button
+  onSubmit(): void {
+    alert('Invitation send successfully');
+  }
+
+  closeForm(): void {
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts:87]: Missing comment

File: src/app/friend/add-friend/add-friend.component.ts [hunk 34]
```diff
@@ -0,0 +1,56 @@
...
+  addNameInput(): void {
+    this.names.push(this.createNameInput());
+  }
+
+  // This method is called when the form is submitted
+  deleteNameInput(index: number): void {
+    this.names.removeAt(index);
+  }
+
+  // This method is called when the user clicks the close button
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts]: Inconsistency in methos comments

File: src/app/friend/add-friend/add-friend.component.ts [hunk 35]
```diff
@@ -0,0 +1,56 @@
...
+    return this.inviteForm.get('names') as FormArray;
+  }
+
+  // This method adds a new name input field to the FormArray
+  addNameInput(): void {
+    this.names.push(this.createNameInput());
+  }
+
+  // This method is called when the form is submitted
+  deleteNameInput(index: number): void {
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts:69]: Inconsistency in methos comments

File: src/app/friend/add-friend/add-friend.component.ts [hunk 36]
```diff
@@ -0,0 +1,56 @@
...
+    });
+  }
+
+  // This method returns the FormArray of names from the inviteForm FormGroup
+  get names(): FormArray {
+    return this.inviteForm.get('names') as FormArray;
+  }
+
+  // This method adds a new name input field to the FormArray
+  addNameInput(): void {
```
Comment (AshaPatel216) [inline src/app/friend/add-friend/add-friend.component.ts:60]: Inconsistency in methos comments

File: src/app/friend/friend-list/friend-list.component.html [hunk 37]
```diff
@@ -0,0 +1,37 @@
+<!--Friend list-->
+<div class="friend-list" >
+    <div class="container" style="overflow-x:auto;">
+        <div class="text-end">
+            <button class="btn btn-light" routerLink="/friend/add-friend">Add</button>
```
Comment (AshaPatel216) [inline src/app/friend/friend-list/friend-list.component.html]: Missing button type

File: src/app/friend/friend-list/friend-list.component.ts [hunk 38]
```diff
@@ -0,0 +1,33 @@
...
+  selector: 'app-friend-list',
+  templateUrl: './friend-list.component.html',
+  styleUrls: ['./friend-list.component.scss']
+})
+export class FriendListComponent {
+
+  constructor(private router: Router){};
+
+  //array of friend
+  friends: Friend[] = [
```
Comment (AshaPatel216) [inline src/app/friend/friend-list/friend-list.component.ts:18]: Inconsistency in methos comments to whole file

File: src/app/group/add-group/add-group.component.html [hunk 39]
```diff
@@ -1,29 +1,66 @@
-<!--Group page-->
-<p>START A NEW GROUP</p>
-<div>
-    <form action="">
-        <div class="group-name">
-            <h1>My group shall be called....</h1>
-            <input type="text" placeholder="Enter Group Name">
+<div class="group">
+  <p class="text-center" style="color: #ffff;">START A NEW GROUP</p>
```
Comment (AshaPatel216) [inline src/app/group/add-group/add-group.component.html]: avoid inline style

File: src/app/body/body.component.html (hunk 18 above)
Comment (1810Nikita) [reply src/app/body/body.component.html]: changes done.

File: src/app/app.component.scss (hunk 19 above)
Comment (1810Nikita) [reply src/app/app.component.scss:31]: changes done.

File: src/app/app.component.scss (hunk 20 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Changes done.

File: src/app/app.component.scss (hunk 21 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Changes done.

File: src/app/dashboard/dashboard.component.html (hunk 22 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.html]: Changes done

File: src/app/dashboard/dashboard.component.html (hunk 23 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.html]: Changes done.

File: src/app/dashboard/dashboard.component.html (hunk 24 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.html]: Changes done.

File: src/app/dashboard/dashboard.component.scss (hunk 25 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.scss]: Changes done.

File: src/app/dashboard/dashboard.component.scss (hunk 26 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.scss]: Changes done.

File: src/app/expenses/expense-list/expense-list.component.html (hunk 27 above)
Comment (1810Nikita) [reply src/app/expenses/expense-list/expense-list.component.html]: Changes done.

File: src/app/friend/friend-list/friend-list.component.scss (hunk 28 above)
Comment (1810Nikita) [reply src/app/friend/friend-list/friend-list.component.scss]: Changes done.

File: src/app/expenses/expense-list/expense-list.component.ts (hunk 29 above)
Comment (1810Nikita) [reply src/app/expenses/expense-list/expense-list.component.ts:75]: Changes done.

File: src/app/expenses/expense-list/expense-list.component.ts (hunk 30 above)
Comment (1810Nikita) [reply src/app/expenses/expense-list/expense-list.component.ts:85]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 31 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 32 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 33 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts:87]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 34 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 35 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts:69]: Changes done.

File: src/app/friend/add-friend/add-friend.component.ts (hunk 36 above)
Comment (1810Nikita) [reply src/app/friend/add-friend/add-friend.component.ts:60]: Changes done.

File: src/app/friend/friend-list/friend-list.component.html (hunk 37 above)
Comment (1810Nikita) [reply src/app/friend/friend-list/friend-list.component.html]: Changes done.

File: src/app/friend/friend-list/friend-list.component.ts (hunk 38 above)
Comment (1810Nikita) [reply src/app/friend/friend-list/friend-list.component.ts:18]: Changes done.

File: src/app/group/add-group/add-group.component.html (hunk 39 above)
Comment (1810Nikita) [reply src/app/group/add-group/add-group.component.html]: Changes done.

File: src/app/app.component.ts [hunk 40]
```diff
@@ -9,17 +16,66 @@ import { Router } from '@angular/router';
...
+      } else {
+        this.showSideNav = true;
+      }
+    });    
   }
-  
+
+  /**
+   * lifecycle hook that is called when the component is initialized
+   * @param none
```
Comment (AshaPatel216) [inline src/app/app.component.ts:52]: If there is no parameter then no need to add the @param

File: src/app/dashboard/dashboard.component.scss [hunk 41]
```diff
@@ -0,0 +1,101 @@
...
+@media only screen and (max-width: 600px) {
+    .grid-container-all-expense {
+      grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+      gap: 5px;
+    }
+}
+
+//on small screen adjust the height of dashboard 
+@media (max-width: 600px) {
+    .dashboard {
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.scss:76]: duplicate media query

File: src/app/dashboard/dashboard.component.scss [hunk 42]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+@media (min-width:600px) and (max-width: 850px) {
+    .dashboard {
+        height: 93.5vh;
+    }
+}
+
+@media (max-width: 600px) {
+    .dashboard-btn {
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.scss]: duplicate media query

File: src/app/sidenav/sidenav.component.scss [hunk 43]
```diff
@@ -0,0 +1,118 @@
+//sidenav style 
+.sidenav {
+  background: #f1e8f1;
```
Comment (AshaPatel216) [inline src/app/sidenav/sidenav.component.scss]: Still the same issues I have found in SCSS files

File: src/app/dashboard/dashboard.component.scss (hunk 41 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.scss:76]: Changes done

File: src/app/dashboard/dashboard.component.scss (hunk 42 above)
Comment (1810Nikita) [reply src/app/dashboard/dashboard.component.scss]: Changes done.

File: src/app/sidenav/sidenav.component.scss (hunk 43 above)
Comment (1810Nikita) [reply src/app/sidenav/sidenav.component.scss]: Changes done.

**Pull request merged**